managing-packages/virtual-environments-and-projects.md

```diff
@@ -178,6 +203,7 @@ What is the general correct order of steps for creating, then activating, then d
 1. `cd project-root-folder`
 1. `python3 -m venv venv`
 1. `source venv/bin/activate`
+1. `pip install -r requirements.txt`
 1. `deactivate`
```

Comment:
GitHub isn't letting me put a comment on a line you didn't modify, but the question should have an extra part added. Instead of 

> What is the general correct order of steps for creating, then activating, then deactivating a virtual environment?

I would suggest

> What is the general correct order of steps for creating, then activating, then installing the dependencies listed in a file, then deactivating a virtual environment?